dvc/command/data_sync.py

```diff
@@ -158,6 +160,12 @@ def add_parser(subparsers, _parent_parser):
         default=False,
         help="Pull cache for subdirectories of the specified directory.",
     )
+    pull_parser.add_argument(
+        "--drop-index",
+        action="store_true",
+        default=False,
+        help="Drop local index for the specified remote cache.",
```

Comment:
`remote cache` -> `remote` (we try to avoid using `cache` with remotes, just to make it less confusing with `.dvc/cache`) ?